.github/workflows/build.yml

```diff
@@ -13,10 +14,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.17
+    - name: Set up Go 1.19
```

Comment:
Hrmmm, 1.19 is no longer supported - are we able to get this to 1.21 or 1.22?

Reply:
Good catch, have updated to 1.22